src/pgduckdb_types.cpp

```diff
@@ -901,13 +902,19 @@ HeapTupleFetchNextColumnDatum(TupleDesc tupleDesc, HeapTuple tuple, HeapTupleRea
                               int att_num, bool *is_null) {
 	HeapTupleHeader tup = tuple->t_data;
 	bool hasnulls = HeapTupleHasNulls(tuple);
+	bool natts = HeapTupleHeaderGetNatts(tup);
```

Comment:
maybe int?

Reply:
Yes, that was it... Clearly time for the weekend...